### PR TITLE
lazygit: 0.8 -> 0.10.6

### PR DESCRIPTION
**Pull request by equirosa**
###### Motivation for this change

Current version of lazygit is slightly outdated.
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @fpletz

### CODE REVIEW

File: pkgs/development/tools/lazygit/default.nix [hunk 1]
```diff
@@ -11,14 +11,14 @@ buildGoPackage rec {
   src = fetchFromGitHub {
     owner = "jesseduffield";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "0zynw5gr96a59x1qshzhhvld883ndf1plnw6l9dbhmff0wcfv6l1";
+    rev = "963c034b481e6676656e981cbc8e2f923cc1ac41";
```
Comment (JohnAZoidberg) [inline pkgs/development/tools/lazygit/default.nix]:
```suggestion
    rev = "v${version}";
```

Please keep it that way. When the package is version 0.10.6 we actually want that.
If you want to add some patches on top that's okay.

But they seem to make new releases very quickly, so we can just update it again, then, to include your favorite patches from master :)

**Review by JohnAZoidberg — CHANGES_REQUESTED** [review]

**Comment by JohnAZoidberg** [comment]
Also please squash the commits together, make sure everything fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and tick that last box.

For next time: It's nice to include a link to the new release and/or changelog in "Motivation" :)

**Comment by equirosa** [comment]
> 
> 
> Also please squash the commits together, make sure everything fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and tick that last box.
> 
> For next time: It's nice to include a link to the new release and/or changelog in "Motivation" :)

Got it! Will fix ASAP :)

**Comment by JohnAZoidberg** [comment]
Why closed? Would you like to update it to [0.11.3](https://github.com/jesseduffield/lazygit/releases/tag/v0.11.3).

**Comment by equirosa** [comment]
> Why closed? Would you like to update it to [0.11.3](https://github.com/jesseduffield/lazygit/releases/tag/v0.11.3).

 It was the first time having to make some changes to a pull request so I thought to make a new one since the version change was different. Kinda new to this...

**Comment by JohnAZoidberg** [comment]
Did you make a new one? Please link to it here.
No, you don't have to make a new one. It's perfectly fine, to continue here. Just use make your changes (even rewriting old commits) and `git push --force`. :)

**Comment by equirosa** [comment]
> Did you make a new one? Please link to it here.
> No, you don't have to make a new one. It's perfectly fine, to continue here. Just use make your changes (even rewriting old commits) and `git push --force`. :)

Oh, ok! Thanks for your help! :smile: 
And here's the other one I made: https://github.com/NixOS/nixpkgs/pull/74895

**Pull request closed without merging**